src/api/migrations.ts

```diff
@@ -145,12 +145,19 @@ export const migrateRepo = async (repoId: string): Promise<void> => {
       )
 
       // Create Repo Document (last db operation)
-      await createRepoDocument(repoId, {
-        paths: repoModification.paths,
-        timestamp: lastGitTime,
-        lastGitHash,
-        lastGitTime: lastGitTime
-      })
+      try {
+        await createRepoDocument(repoId, {
+          paths: repoModification.paths,
+          timestamp: lastGitTime,
+          lastGitHash,
+          lastGitTime: lastGitTime
+        })
+      } catch (err) {
+        // Silence conflict errors
```

Comment:
While the API call should not throw, I think it's worth at least putting something in the logs. Put a console.log here so we know a conflict happened since that should be rare

Reply:
Good idea.